BRTransaction.h

```diff
@@ -34,7 +34,7 @@
 extern "C" {
 #endif
 
-#define TX_FEE_PER_KB        1000ULL     // standard tx fee per kb of tx size, rounded up to nearest kb
+#define TX_FEE_PER_KB        191ULL     // standard tx fee per kb of tx size, rounded up to nearest kb
```

Comment:
This is wrong at face value, but there may be a deeper issue in the code using it. (Should I spend time investigating that further?)

Reply:
This is the change we are the most unsure about, and yes, can you spend some time getting the right value, our objective was to be able to send `1 sat / kb` when breadwallet allowed a minimum of `5`.

@docbtc mentioned me on his tests he saw a difference in the fees we're sending. We want to make this correct, so if you can help us would be great.

Reply:
The right value is definitely 1000. The problem is with the code using it.